### ISSUE TITLE
QueueUrl >> make configurable

### ISSUE THREAD

**Issue by luispabon** [p4tin/goaws#67]
I'm integrating the docker image into a docker-compose environment. I'm having trouble as SQS queues coming back from getqueueurl are all pointing to localhost, which is not correct in this environment and won't work as obviously my application is running on a different container.

**Comment by p4tin** [comment]
Can you give me some info about your environment?  
 - Mac/Win/Linux? (VirtualBox/Docker Beta?)
 - I assume you do a GetQueueURL and then try to access the Queues using the URL from the response?

**Comment by giorgiosironi** [comment]
I am experiencing a form of this, it seems queue retain in their URLs the host they were created with:
```
[16:49:30][elife@dev--annotations:~]$ aws --endpoint http://localhost:4100 sqs create-queue --queue-name=hello1                            
{
    "QueueUrl": "http://localhost:4100/queue/hello1"
}
[16:49:37][elife@dev--annotations:~]$ aws --endpoint http://localhost:4100 sqs list-queues                                                 
{
    "QueueUrls": [
        "http://localhost:4100/queue/hello1"
    ]
}
[16:49:40][elife@dev--annotations:~]$ aws --endpoint http://annotations--vagrant:4100 sqs create-queue --queue-name=hello2                 
{
    "QueueUrl": "http://annotations--vagrant:4100/queue/hello2"
}
[16:49:48][elife@dev--annotations:~]$ aws --endpoint http://annotations--vagrant:4100 sqs list-queues                                      
{
    "QueueUrls": [
        "http://localhost:4100/queue/hello1", 
        "http://annotations--vagrant:4100/queue/hello2"
    ]
}
```

I had specified `Host` in the configuration, but it seems to be ignored:
```
[16:49:16][elife@dev--annotations:~]$ docker run -p 4100:4100 -v $(pwd)/goaws/conf:/conf pafortin/goaws -config /conf/goaws.yaml -debug
{"level":"warning","msg":"Loading config file: /conf/goaws.yaml","time":"2018-01-19T16:49:18Z"}
{"level":"warning","msg":"GoAws listening on: 0.0.0.0:4100","time":"2018-01-19T16:49:18Z"}
{"level":"info","msg":"Listing Queues","time":"2018-01-19T16:49:29Z"}
{"level":"info","msg":"Creating Queue: hello1","time":"2018-01-19T16:49:37Z"}
{"level":"info","msg":"Listing Queues","time":"2018-01-19T16:49:40Z"}
{"level":"info","msg":"Creating Queue: hello2","time":"2018-01-19T16:49:48Z"}
{"level":"info","msg":"Listing Queues","time":"2018-01-19T16:49:53Z"}
^C[16:51:07][elife@dev--annotations:~]$ cat goaws/conf/goaws.yaml 
Local:
  Host: goaws
  Port: 4100
  #Queues:
  #  - Name: queue1
```

**Comment by giorgiosironi** [comment]
I think this problem is limited to queues that are dynamically created. If I put queues in the configuration, I get:
```
$ aws --endpoint http://annotations--vagrant:4100 sqs list-queues
{
    "QueueUrls": [
        "http://goaws:4100/queue/annotations--dev"
    ]
}
```
which is what I expected, picking up `Host` from the configuration.

**Comment by giorgiosironi** [comment]
In particular, it should come from this:
https://github.com/p4tin/goaws/blob/0f96ff53888fdfaa11c0b50d28c6c3f8d9e9ec5e/app/gosqs/gosqs.go#L90
which reads the request rather than the configuration.

**Comment by p4tin** [comment]
@giorgiosironi nice catch I am re-opening this issue

**Comment by p4tin** [comment]
Fixed in version 1.0.1 - docker image 1.0.1
